### PR TITLE
feat(ip): Complete help for unknown subcommands

### PR DESCRIPTION
**Pull request by yedayak**

### CODE REVIEW

File: completions/ip [hunk 1]
```diff
@@ -417,10 +417,14 @@ _comp_cmd_ip()
                     ;;
                 *)
                     ((cword == subcword)) &&
-                        _comp_compgen -- -W 'state policy monitor'
+                        _comp_compgen -- -W 'state policy monitor help'
```
Comment (akinomyoga) [inline completions/ip:420]: The other cases seem to have `help` at the beginning of the list.

File: completions/ip [hunk 2]
```diff
@@ -405,7 +405,7 @@ _comp_cmd_ip()
                     fi
                     ;;
                 *)
-                    _comp_compgen -- -W 'show'
+                    _comp_compgen -- -W 'show help'
```
Comment (akinomyoga) [inline completions/ip:408]: The other cases seem to check `((cword == subword))`, but this branch doesn't check that. Do you think there is a reason? For example, is there a use case of `ip netconf <something>... show` or `ip netconf <something>... help`?

File: completions/ip (hunk 1 above)
Comment (yedayak) [reply completions/ip]: Changed it, also in netconf

File: completions/ip (hunk 2 above)
Comment (yedayak) [reply completions/ip]: No, that was an oversight. `ip netconf <something>... help` doesn't do anything useful. Fixed it

**Review by akinomyoga — APPROVED** [review]
Thanks!

**Pull request merged**